src/patternfly/utilities/BoxShadow/box-shadow.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="{{#if box-shadow--modifier}} {{box-shadow--modifier}}{{/if}}"
+  {{#if box-shadow--attribute}}
```

Comment:
picky but should this be `box-shadow--attributes`?

Reply:
lol, https://github.com/patternfly/patternfly-next/issues/499
I remember consensus being singular form

Reply:
doh! 👍